api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -177,9 +177,13 @@ echo "Inserting concept table data ... "
 bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
 "INSERT INTO \`$WORKBENCH_PROJECT.$WORKBENCH_DATASET.concept\`
 (concept_id, concept_name, domain_id, vocabulary_id, concept_class_id, standard_concept,
-concept_code, count_value, prevalence, source_count_value)
+concept_code, count_value, prevalence, source_count_value, synonyms)
+with concept_synonyms as
```

Comment:
Any reason not to just inline this in the SELECT subquery below?

Reply:
I was thinking of some concepts which do not have the synonyms listed.

Reply:
Not sure why that matters? Anyway, so long as the query works, this is fine.